package.json

```diff
@@ -46,12 +46,13 @@
     ],
     "testPathIgnorePatterns": [
       "/node_modules/",
+      "<rootDir>/integration-tests/create-react-app",
       "<rootDir>/packages/react-percy-webpack/src/entry/__tests__/test-project/"
     ]
   },
   "lint-staged": {
     "*.js": [
-      "eslint --fix --color --no-ignore",
```

Comment:
haven't tried the pattern, but a pattern with `!.` is the same as `--no-ignore` in our case, no?